### DISCUSSION TITLE
Find the Custom Module location

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15593] [How to] [answered]
Hi,

I created one custom module and i can create the records using Quick Create button. 

But I am unable to find the Custom module in the left side panel. Could you please helpe me to find the location of custom modules in the CRM.

Thanks & regards,
Rajasekaran.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
after creating the module, you need to add it to the menu because it will not be visible in the auto menu.
https://gitdeveloper.yetiforce.com/index.php?module=Menu&view=Index&parent=Settings

**Reply by Ilamparithi-Tamizhanangu** [reply]
Thank you :)

Now i can able to add the module to the menu.
